security/pkg/server/ca/server_test.go

```diff
@@ -179,6 +179,7 @@ func TestSign(t *testing.T) {
 			port:           8080,
 			authorizer:     c.authorizer,
 			authenticators: c.authenticators,
+			monitoring:     newMonitoringMetrics(),
```

Comment:
Should we add some test for monitoring if monitoring dependency can be arranged? Could do it later.

Reply:
Yes, I will add the unit tests later.

Reply:
We should probably add a dashboard for Citadel to the existing dashboards and add an e2e test to handle while we are at it.

Reply:
For the dashboard, what do we need to change except the yaml file?

Reply:
we would need to build a new dashboard like the mixer and pilot ones in: https://github.com/istio/istio/tree/master/addons/grafana/dashboards

And then add a test that validates the queries in that dashboard work.

I can help with that after this stuff goes in.